functions/powersOfTwo/powersOfTwo.js

```diff
@@ -0,0 +1,12 @@
+export const powersOfTwo = (n) => {
+  const arr = []
+  
+  for (let i=0; i<=n; i++) {  
```

Comment:
`  for (let index = 0; index <= number; index++) { } `
or
`  for (let count = 0; count <= number; count++) { } `